cmd/kubelet/app/server.go

```diff
@@ -95,6 +96,7 @@ type KubeletServer struct {
 	TLSPrivateKeyFile              string
 	CertDirectory                  string
 	NodeStatusUpdateFrequency      time.Duration
+	ChaosChance                    float64
```

Comment:
I think there is a small set of folks that would run Chaos in production either as canaries or continuous test, but I agree with the sentiment and will add comments to them.

----- Original Message -----
> > @@ -95,6 +96,7 @@ type KubeletServer struct {
> >  	TLSPrivateKeyFile              string
> >  	CertDirectory                  string
> >  	NodeStatusUpdateFrequency      time.Duration
> > +	ChaosChance                    float64
> 
> My concern is with "spinkling of test code" throughout the codebase. I'd
> prefer to have them separated out in an easily discernible way, in case we
> wish to rip them out or refactor them in the future. At the least, please
> comment.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6729/files#r28284595